domain/src/main/kotlin/no/nav/su/se/bakover/domain/vilkår/OppholdIUtlandetVilkår.kt

```diff
@@ -181,6 +202,27 @@ data class VurderingsperiodeOppholdIUtlandet private constructor(
                 begrunnelse = begrunnelse,
             ).right()
         }
+
+        fun Nel<VurderingsperiodeOppholdIUtlandet>.slåSammenVurderingsperioder(): Nel<VurderingsperiodeOppholdIUtlandet> {
+            val slåttSammen = this.sortedBy { it.periode.fraOgMed }
+                .fold(mutableListOf<MutableList<VurderingsperiodeOppholdIUtlandet>>()) { acc, oppholdIUtlandet ->
+                    if (acc.isEmpty()) {
+                        acc.add(mutableListOf(oppholdIUtlandet))
+                    } else if (acc.last().sisteFormueperiodeErLikOgTilstøtende(oppholdIUtlandet)) {
```

Comment:
Dette ser litt suspekt ut? `sisteFormueperiodeErLikOgTilstøtende`

Reply:
Det er vel denne funksjonen det står på for å lage noe felles. Foreslår at vi tar det i neste PR